src/DocumentPicker.Plugin/Shared/DocumentConverterService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using DocumentConverter.Plugin.Shared.Picker;
+using DocumentConverter.Plugin.Shared.StreamProvider;
+using PdfToSvg;
+
+namespace DocumentConverter.Plugin.Shared
+{
+    public class DocumentConverterService : IDocumentConverterService
+    {
+
+        private readonly IFilePicker _filePicker;
+        private readonly ICustomStreamProvider _customStreamProvider;
+
+        public DocumentConverterService(IFilePicker filePicker, ICustomStreamProvider customStreamProvider)
+        {
+            _filePicker = filePicker;
+            _customStreamProvider = customStreamProvider;
+        }
+
+        public async Task<PdfFileInfo> ConvertPdfToSvgAsync(string filePath, string outputDir, int page = 1, CancellationToken cancellationToken = default)
```

Comment:
could you please use 0-based indexes?
`int pageIndex = 0` makes much more sense to a developer.